<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nPython</class>
 <widget class="QMainWindow" name="nPython">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
    <property name="margin">
     <number>4</number>
    </property>
    <item row="1" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="neutrinoSave" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="loadScript">
      <property name="toolTip">
       <string>Load script</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/neutrino.qrc">
        <normaloff>:/loadPref</normaloff>:/loadPref</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="runScript">
      <property name="toolTip">
       <string>Run script</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/neutrino.qrc">
        <normaloff>:/gearPy</normaloff>:/gearPy</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="scriptFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Script filename</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="neutrinoSave" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/neutrino.qrc"/>
 </resources>
 <connections/>
</ui>
